src/kvilib/config/KviBuildInfo.cpp

```diff
@@ -88,11 +88,11 @@ const QStringList feature_list{
 #endif
 #ifdef COMPILE_ENCHANT_SUPPORT
 	"Enchant",
-#endif
-#ifdef COMPILE_ENABLE_GTKSTYLE
-	"GTK",
 #endif
 	"Qt5",
```

Comment:
maybe put Qt5 to #else branch?

Reply:
I kept Qt5 since some scripts may need it, eg. `/echo $features(Qt5)` to determine if  some features only available since Qt5 are present

Reply:
Okay, but a comment which explains that would be useful

Reply:
Done in 03a8df16588ab42b4f4a070fc665e7ccf512e89d